src/policy/sft.rs

```diff
@@ -101,6 +101,27 @@ pub trait SFT {
         object: ObjectReference,
         worker: GCWorkerMutRef,
     ) -> ObjectReference;
+
+    fn debug_get_object_info(&self, object: ObjectReference) -> String {
+        format!(
+            "{}: This policy does not implement debug_get_object_info. Global info: {}",
+            self.name(),
+            crate::policy::sft::debug_get_object_global_info(object)
+        )
+    }
+}
+
+pub(crate) fn debug_get_object_global_info(_object: ObjectReference) -> String {
+    #[allow(unused_mut)]
+    let mut ret = String::new();
+    #[cfg(feature = "vo_bit")]
+    {
+        ret += &format!(
+            "vo bit = {}",
+            crate::util::metadata::vo_bit::is_vo_bit_set(_object)
```

Comment:
`is_vo_bit_set` doesn't check if the metadata is mapped or not.  We should call `is_mmtk_object` or equivalently `is_vo_bit_set_for_addr` instead.  And we should actually check it in `crate::mmtk::mmtk_debug_print_object_info` when converting the `Address` to `ObjectReference`.

Reply:
I added a check to check if the object is mapped or not before forwarding to SFT. If the object is mapped, we can assume all the metadata can be accessed.

Reply:
If the object is mapped, but some side metadata is not mapped, then it is a bug in MMTk core. We do not expect the function to work properly, and we do not expect the function can help us figure out the bug either.